secret-engine/src/main/java/com/sequenceiq/cloudbreak/service/secret/vault/AbstractVaultEngine.java

```diff
@@ -56,4 +58,12 @@ protected Gson gson() {
     }
 
     protected abstract Class<E> clazz();
+
+    public List<String> listSecretPaths(String path) {
+        return Collections.EMPTY_LIST;
+    }
+
+    public void cleanup(String path) {
```

Comment:
this should be abstract so if a new engine is added then who adds it would be aware of this should be implemented

Reply:
In that case, i will have these dummy implementations in V1 engine.